### PR TITLE
Fix alignment of NxTextInput and NxFilterInput when adjacent - RSC-599

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-599

### CODE REVIEW

File: lib/src/components/NxFilterInput/NxFilterInput.scss [hunk 1]
```diff
@@ -30,15 +30,10 @@
...
-  .nx-text-input__box {
-    padding-bottom: 0;
-  }
-
-  .nx-text-input__input {
-    padding: 0 0 8px 0;
-  }
-
-  .nx-icon--filter-icons {
-    margin-bottom: 8px;
```
Comment (rpokorny) [inline lib/src/components/NxFilterInput/NxFilterInput.scss:42]: These deleted styles were basically just doing the same thing the nx-text-input styles do but in a different way.  Deleting them made no difference and made it simpler to understand and solve the alignment problem.

**Comment by glenhunter** [comment]
Unfortunately Safari is not playing along: 
Screen Shot 2021-06-17 at 12 18 19 PM

and Chrome seems to be off by just a pixel: 
Screen Shot 2021-06-17 at 12 18 10 PM

**Comment by rpokorny** [comment]
I can reproduce that it's wrong in safari but I just double checked that chrome is perfect for me

**Comment by rpokorny** [comment]
Fix for safari: bf7c786e

**Comment by glenhunter** [comment]
Right now there is no default spacing between the form fields, I remember that was mentioned in WW this week but I can't recall the outcome and it isn't recorded anywhere. @dturner32 ?

**Comment by rpokorny** [comment]
> Right now there is no default spacing between the form fields, I remember that was mentioned in WW this week but I can't recall the outcome and it isn't recorded anywhere. @dturner32 ?

I don't think we came to much of a conclusion on it since the one place that currently needs this has spacing needs that wouldn't be what we'd want more generally.

**Comment by glenhunter** [comment]
> > Right now there is no default spacing between the form fields, I remember that was mentioned in WW this week but I can't recall the outcome and it isn't recorded anywhere. @dturner32 ?
> 
> I don't think we came to much of a conclusion on it since the one place that currently needs this has spacing needs that wouldn't be what we'd want more generally.

I thought we said something like just `4px`

**Comment by rpokorny** [comment]
> I thought we said something like just 4px

I know we talked _about_ that, but in my recollection we didn't agree to go with it.

**Comment by dturner32** [comment]
> Right now there is no default spacing between the form fields, I remember that was mentioned in WW this week but I can't recall the outcome and it isn't recorded anywhere. @dturner32 ?



> > I thought we said something like just 4px
> 
> I know we talked _about_ that, but in my recollection we didn't agree to go with it.

if I recall, we landed on 16px being the text input and the filter input

**Comment by rpokorny** [comment]
> if I recall, we landed on 16px being the text input and the filter input

99a778a4

**Review by glenhunter — APPROVED** [review]
Safari issues are resolved, looks good.

**Pull request merged**